README.md

```diff
@@ -75,6 +75,11 @@ const searchClient = instantMeiliSearch(
 )
 ```
 
+### Options
+
+- `Host` - URI of Meilisearch instance
```

Comment:
```suggestion
- `Host` - URL of Meilisearch instance
```

Why do you think we should say `URI` and not `URL` ?

Reply:
That was a misnomer, this should say URL :+1: